src/components/player/media/Gif.tsx

```diff
@@ -18,6 +18,7 @@ export default function Gif(props: MediaProps) {
     height,
     detailsVisible,
     isFullscreen,
+    onBack,
```

Comment:
No need to unpack it here if it won't be used. This actually generates a warning.